diff --git a/timeline.module b/timeline.module
index 1ef1a4f..a63baef 100644
--- a/timeline.module
+++ b/timeline.module
@@ -184,7 +184,7 @@ function timeline_validate_library() {
       '/timeline_ajax/scripts/signal.js',
     );
     // check if library is available 'sites/all/libraries/simile_timeline'
-    if (is_dir($timeline_path)) {
+    if (file_exists($timeline_path)) {
       $messages = '';
       foreach ($timeline_library_files as $file) {
         $file = $timeline_path . $file;
