diff --git a/plugins/date_sources/date_field.inc b/plugins/date_sources/date_field.inc
index 93b4154..c999321 100644
--- a/plugins/date_sources/date_field.inc
+++ b/plugins/date_sources/date_field.inc
@@ -9,7 +9,7 @@ $plugin = array(
   'name' => t('Date field'),
   'callback' => 'timeline_date_source_date_field_conversion',
   'handler_name' => 'views_handler_field_field',
-  'field_type' => 'date',
+  'field_type' => 'datetime',
 );
 
 /**
